diff --git a/components/crash/content/app/crashpad_linux.cc b/components/crash/content/app/crashpad_linux.cc
index 05767fe2a26e16a0152ba419ed38a944f919cbde..1488dd00a7877f78955cb96396c73b8ecc1a734d 100644
--- a/components/crash/content/app/crashpad_linux.cc
+++ b/components/crash/content/app/crashpad_linux.cc
@@ -143,6 +143,8 @@ bool BuildHandlerArgs(base::FilePath* handler_path,
 
 #if defined(GOOGLE_CHROME_BUILD) && defined(OFFICIAL_BUILD)
   *url = "https://clients2.google.com/cr/report";
+#elif defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
+  *url = "https://laptop-updates.brave.com/1/bc-crashes";
 #else
   *url = std::string();
 #endif
